src/endpoints/user/deleteUser.ts

```diff
@@ -0,0 +1,19 @@
+import {IncomingMessage, ServerResponse} from 'http'
+import { UserService } from '../../userService';
+import { getUserIdFromRequest } from '../../utils';
+import { PATHS } from '../../constants';
+
+export function deleteUserByIdRoute(req: IncomingMessage, res: ServerResponse) {
+    const userService = UserService.getInstance();
+    const userId: number = getUserIdFromRequest(req, res, PATHS.userId);
+    const users = userService.deleteUser(userId)
+
+    if (users?.length) {
+        res.statusCode = 200;
+        res.end(JSON.stringify(users));
```

Comment:
We can send in response deleted user's object or just his id, I guess no need to send all users to the client